packages/graalvm/src/main/kotlin/elide/runtime/gvm/internals/vfs/EmbeddedGuestVFSImpl.kt

```diff
@@ -376,12 +376,25 @@ internal class EmbeddedGuestVFSImpl private constructor (
     }
   }
 
+  private fun fixAttributeTypes(map: MutableMap<String, Any>): MutableMap<String, Any> {
+    return map.apply {
+      if (containsKey("ino")) {
+        val ino = this["ino"]
+        if (ino is Int) {
+          this["ino"] = ino.toLong()
+        }
+      }
+    }
+  }
```

Comment:
Recasts `ino` as a `Long`, which is the expected type for `TruffleFile`; JimFS produces an `Int`